src/ray/core_worker/common.h

```diff
@@ -87,7 +88,8 @@ struct ActorCreationOptions {
         name(name),
         is_asyncio(is_asyncio),
         placement_options(placement_options),
-        placement_group_capture_child_tasks(placement_group_capture_child_tasks){};
+        placement_group_capture_child_tasks(placement_group_capture_child_tasks),
+        override_worker_env(override_worker_env){};
```

Comment:
```suggestion
        override_worker_env(override_worker_env) {};
```

Reply:
Actually `ci/travis/format.sh` automatically changes it back to the original format without the space, which is odd and inconsistent with the rest of the file.  Do you know a good workaround?  I've just left it in the original format for now so I can push.